### DISCUSSION TITLE
How to froce / in Nuxt 3

### DISCUSSION THREAD

**Discussion by DidoMarchet** [nuxt/nuxt#25325] [General]
Hi,
maybe I missed something in the doc but I'm not able to allow / on all routes.

In Nuxt2 the router configuration provides to redirect from /xxx to /xxx/ but in Nuxt 3 I add:
```
 experimental: {
    defaults: {
      nuxtLink: {
        trailingSlash: 'append',
      },
    },
}
``` 
and from the @nuxt/seo doc 
```
site: {
    trailingSlash: true
  }
``` 
but I would like to directly redirect if I write /xxx witout workaround such as add the / in a router middleware.

Any advice would be appreciated!

Thanks and kind regards,

Davide

**Comment by xjccc** [comment]
[docs](https://nuxt.com/docs/api/components/nuxt-link#overwriting-defaults)

**Comment by DidoMarchet** [comment]
Hi thanks but I already did it. In the previous versions of Nuxt there was a router attribute, trailingslash https://v2.nuxt.com/docs/configuration-glossary/configuration-router/#trailingslash, when set to true it appended / to all routes, sitemap, canonical etc. In Nuxt3 no. 
Btw I decided to remove all slash from routes.
